### ISSUE TITLE
RFE: add ability to modify token labels

### ISSUE THREAD

**Issue by leathersole** [freeotp/freeotp-ios#151]
On FreeOTP iOS v2.0 Beta, $subject is removed. That almost works on my end, except for aws root accounts.
The label is not so readable. If I register multiple aws accounts, I wouldn't be able to distinguish between them.
So the feature is needed.
![aws](https://user-images.githubusercontent.com/93586/82963903-fb02dd80-9ffe-11ea-9515-48509e7995be.jpg)

**Comment by ctueck** [comment]
I agree, the ability to manually edit accounts, especially the labels, would be welcome. I have one instance where the account added by QR code shows no info except my email (username of that service):

![image](https://user-images.githubusercontent.com/37637883/87422783-8816eb80-c5d9-11ea-9138-03c0e94c4195.png)

I assume that's something badly configured on the side of the site I am logging in to, but it would help if one could just fix it manually.

It seems issue #186 might be related, as the general ability to manually enter and edit would probably solve that as well.

**Comment by ctueck** [comment]
PS: Thanks for the great app generally! It's highly welcome to have a simple and open-source solution for that.
